narwhal/primary/src/state_handler.rs

```diff
@@ -57,23 +60,45 @@ impl StateHandler {
                 rx_state_handler,
                 tx_reconfigure,
                 last_committed_round: 0,
+                tx_commited_own_headers,
                 network,
             }
             .run()
             .await;
         })
     }
 
-    async fn handle_sequenced(&mut self, certificate: Certificate) {
+    async fn handle_sequenced(&mut self, round: Round, _certificates: Vec<Certificate>) {
         // TODO [issue #9]: Re-include batch digests that have not been sequenced into our next block.
```

Comment:
can this TODO be removed?

Reply:
Ah yes, the whole point is to address this issue (which was one of the earlier ones, and in the old narwhal repo)